packages/core/src/components/TagsInput/TagsInput.tsx

```diff
@@ -676,7 +649,6 @@ export const HvTagsInput = ({
               }}
               inputRef={inputRef}
               $singleLine={!multiline}
-              $isTagSelected={isTagSelected}
               {...others}
             />
           </StyledInputListItem>
```

Comment:
Classes inputExtension, suggestionsContainer, and suggestionList missing for the suggestion component.